src/proto/api_messages_test.js

```diff
@@ -15,17 +15,19 @@
  */
 
 import {
-    AnalyticsContext,
-    AnalyticsRequest,
-    AnalyticsEvent,
-    deserialize} from './api_messages';
+  AnalyticsContext,
```

Comment:
unnecessary indent

Reply:
I think the issue here is that it was over indented before and I have unindented it to respect the 2 spaces rule